lib/sqlalchemy/dialects/oracle/cx_oracle.py

```diff
@@ -78,6 +78,11 @@
 
 * ``coerce_to_decimal`` - see :ref:`cx_oracle_numeric` for detail.
 
+* ``text_encoding_errors`` - set the text encoding to use in case of a
```

Comment:
let's call it "encoding_errors" to more closely match cx_oracle's name, and also link to cx_Oracle's documentation here since it's their option.    Also needs a note that under Python 2 we are using it in our own decoder.